<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Kevin8" ignored="0" tests="4" failures="1" timestamp="2021-08-09T20:00:20 EEST" time="0.094" errors="0">
  <testcase name="testIsRegistered" time="0.0" classname="Test"/>
  <testcase name="testSumm" time="0.0" classname="Test"/>
  <testcase name="testUserAge" time="0.0" classname="Test">
    <failure type="java.lang.AssertionError" message="Something wrong with test data expected [31] but found [30]">
      <![CDATA[java.lang.AssertionError: Something wrong with test data expected [31] but found [30]
at Test.testUserAge(Test.java:31)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- testUserAge -->
  <testcase name="testUserName" time="0.0" classname="Test"/>
</testsuite>
